include/glow/Base/Type.h

```diff
@@ -472,6 +472,14 @@ inline ElemKind getScaleOffsetElemKindFromFused(ElemKind e) {
 std::pair<float, float> getQuantizedValueRange(float scale, int32_t offset,
                                                ElemKind elementType);
 
+/// \returns the number of values associated to a quantized type given
+/// \p elementType (e.g. 256 for Int8QTy).
+size_t getQuantizedValueCount(ElemKind elementType);
+
+/// \returns the floating point value step associated to a quantized type given
+/// \p scale, \p offset, and \p elementType.
+float getQuantizedValueStep(float scale, int32_t offset, ElemKind elementType);
+
```

Comment:
nit: These aren't used aside from inside Type, right? Why not just add them to `Type.{cpp,h}`

Reply:
Currently these functions are available both outside and inside (as members of) the ```Type``` class (similar to the `getQuantizedValueRange`). Not sure I understand the request ... you mean we should only have them as class members?

Reply:
Sorry -- yes, I meant right now you forward to them inside `Type:: getQuantizedValueCount` and `Type::getQuantizedValueStep`, but if they're not used outside of the Type class then they could only be class members. Not a huge deal, just seemed unnecessary.

Reply:
I kept the two functions only inside the Type class. Done!